gix/tests/gix/revision/spec/from_bytes/regex.rs

```diff
@@ -95,20 +95,38 @@ mod find_youngest_matching_commit {
     fn regex_matches() {
         let repo = repo("complex_graph").unwrap();
 
-        assert_eq!(
-            parse_spec(":/mes.age", &repo).unwrap(),
-            Spec::from_id(hex_to_id("ef80b4b77b167f326351c93284dc0eb00dd54ff4").attach(&repo))
-        );
+        // On the CI linux workflow, archived baseline aren't used - instead fixtures will be re-evaluated.
```

Comment:
I just noticed that this wording is slightly misleading because the `test-fast` workflow uses archived baselines on all three tested platforms including Linux--this had at one time not been the case, but as far as I know it has been the case ever since the committed archives were moved out of LFS--while really the most relevant distinction is between the `test` and `test-fast` workflows, where the `test` workflow re-evaluates fixtures.

```suggestion
        // On the full linux CI `test` workflow, archived baseline aren't used - instead fixtures will be re-evaluated.
```

I'm not sure what the best wording is, but maybe that slightly improves clarity.